1_fetch/src/get_nwis_data.R

```diff
@@ -1,35 +1,19 @@
+#' Getting NWIS Data 
+#'
+#' @param site_num character, 4 different site_num: "01427207", "01432160", "01436690", "01466500"
+#' @param file_out csv, csv exported for each designated site_num
+#' @param parameterCd 
```

Comment:
The `parameterCd` is the parameter code corresponding to the type of data queried.